db/seeds.rb

```diff
@@ -5,3 +5,59 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Person Seeder
+people = []
+5.times do |i|
+  person = Person.create(
+    first_name: 'Person',
+    last_name: "#{i}",
+    email: "person.#{i}@example.com",
+    date_of_birth: Date.new(1970,1,1),
+    password: 'abc123',
+    password_confirmation: 'abc123',
+    gender: 'Seed'
+  )
+
+  people.push(person)
```

Comment:
I don't think we need to be story these in an array. Later on when you want to select a random user you could do something like:

```ruby
person_id: Person.pluck(:id).sample
```

This logic can apply to all of the seeds.